frontend/components/dashboard/chart/DashboardChartSummaryFilter.vue

```diff
@@ -13,10 +13,11 @@ import { getGroupLabel } from '~/utils/dashboard/group'
 
 const { t: $t } = useTranslation()
 
+const validatorDashboardStore = useValidatorDashboardStore()
 const {
-  hasAbilityCharthistory,
-  overview,
```

Comment:
this could also be part of line `16`, as `validatorDashboarStore.hasAbilityChartHistory` would work too